packages/expo-payments-stripe/README.md

```diff
@@ -21,6 +21,29 @@ For bare React Native projects, you must ensure that you have [installed and con
 expo install expo-payments-stripe
 ```
 
+- Include the config plugin in your `app.config.js` or `app.json`. Add the following props:
+  - `scheme` is the redirect URI that's used for returning to the app after an external payment. This should not match one of your existing schemes as it'll redirect to a special Activity on Android.
```

Comment:
might be better to give this a more specific name to make it clear in code that users shouldn't just pick a scheme that they're already using